### PR TITLE
feat: added logout page

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

File: screens/logout.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                <a href="signup.html" class="btn-small margin-lr">Signup</a>
+
+                <div class="badge-container margin-lr">
+                    <a href="wishlist.html"><img
+                            src="https://img.icons8.com/external-flatart-icons-outline-flatarticons/30/000000/external-like-instagram-flatart-icons-outline-flatarticons.png" />
+                        <span class="badge-span">6</span></a>
+                </div>
+
+                <div class="badge-container margin-lr">
+                    <a href="cart.html"><img src="https://img.icons8.com/ios/30/000000/add-shopping-cart--v1.png" />
```
Comment (pradipgharge) [inline screens/logout.html:37]: alt missing for the image.

File: screens/logout.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
```
Comment (malaykhakhar) [inline screens/logout.html]: Title would be logout

**Review by pradipgharge — APPROVED** [review]
Just add the alt for the image, the rest looks good to merge!

**Review by malaykhakhar — APPROVED** [review]
Rest of the code looks good. You can merge it.

File: screens/logout.html (hunk 1 above)
Comment (eakansh125) [reply screens/logout.html:37]: Done

File: screens/logout.html (hunk 2 above)
Comment (eakansh125) [reply screens/logout.html]: Right thanks

**Pull request merged**